version.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
Probably better than maintaining it in two places, but what is version.json used for? I typically set __version__ in __init__.py and parse it in setup.py.

Reply:
https://github.com/jquast/blessed/blob/master/docs/conf.py#L102-L105

Reply:
Here's what I do for enlighten. Same for most of the packages I maintain.
https://github.com/Rockhopper-Technologies/enlighten/blob/master/setup_helpers.py#L21-L32
https://github.com/Rockhopper-Technologies/enlighten/blob/master/doc/conf.py#L64
https://github.com/Rockhopper-Technologies/enlighten/blob/master/setup.py#L29